cmd/admin-subnet-health.go

```diff
@@ -161,11 +174,85 @@ func mainAdminHealth(ctx *cli.Context) error {
 	}
 
 	if clusterHealthInfo.GetError() != "" {
-		console.Println(warnText("unable to obtain health information:"), clusterHealthInfo.GetError())
+		console.Println(warnText("Unable to obtain health information: "), clusterHealthInfo.GetError())
 		return nil
 	}
 
-	return tarGZ(clusterHealthInfo, aliasedURL)
+	filename := fmt.Sprintf("%s-health_%s.json.gz", filepath.Clean(aliasedURL), time.Now().Format("20060102150405"))
+	error := tarGZ(clusterHealthInfo, filename)
+	if error != nil {
+		console.Println(warnText("Unable to create health report file: "), error.Error())
+		return nil
+	}
+
+	license := ctx.String("license")
+	if len(license) > 0 {
+		error = uploadHealthReport(filename, license, ctx.Bool("dev"))
+		if error != nil {
+			console.Println(warnText("Unable to upload health report to Subnet: "), error.Error())
+		}
+	}
+	return nil
+}
+
+func uploadHealthReport(filename string, license string, dev bool) error {
+	url := subnetUploadURL(filename, license, dev)
+	req, err := subnetUploadReq(url, filename)
+	if err != nil {
+		return err
+	}
+
+	client := &http.Client{}
+	resp, herr := client.Do(req)
+	if herr != nil {
+		return herr
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode == http.StatusOK {
+		console.Infoln("Health data uploaded to Subnet")
```

Comment:
```suggestion
		console.Infoln("MinIO Health data was successfully uploaded to Subnet")
```

It would nice to print the `subnet` URL to view the health report  - this should be returned as part of the response to the successful health report as "Location" header in the response IMO.

Reply:
Actually have updated this PR only for this.